kickstarter/android-oss
# Live backed project reward states

# What ❓
- Renamed `button_manage_pledge` to `button_pledge_manage` to match the naming convention of the other colors.
- For a live backed project:
  - Backed reward:
    - Your selected reward is focused when you open the rewards sheet.
    - The pledge button is blue and says `Manage your pledge`.
    - The checkmark is blue with a blue background.
  - Other available rewards:
    - The pledge button is green and says `Select this instead`.
- Added `RewardUtils.checkmarkBackgroundDrawable` that returns the drawable resource ID of the check background based on project status.
- Updated the output `backedRewardPosition` in `RewardFragmentViewModel` to also emit when the project is live.

# How to QA? 🤔
View a live project that you've backed (check your profile). (Note: The PR does not include any work in the `PledgeFragment`.)

# Story 📖
[Trello Story](https://trello.com/c/qupuDjcL/1413-reward-states-manage-your-pledge-backer-live-project)

# See 👀
![device-2019-06-25-144802 2019-06-25 14_48_28](https://user-images.githubusercontent.com/1289295/60124767-5a55cf00-9758-11e9-8192-52b613f92aec.gif)
